com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/archive/ManifestBuilder.java

```diff
@@ -325,6 +108,22 @@ public ManifestBuilder(boolean outputManifestHash) {
         this.outputManifestHash = outputManifestHash;
     }
 
+    public static ManifestBuilder create() {
+        return create(false);
+    }
+
+    public static ManifestBuilder create(boolean outputManifestHash) {
```

Comment:
Helper to create a ManifestBuilder with correct cryptographic algorithms. We need to use the same resource hash algorithm in both ArchiveBuilder and in the ResourceDigestCache.

Reply:
Is it compatible with [extension-resource-encryption](https://github.com/defold/extension-resource-encryption) ?

Reply:
Yes, the encryption is a separate step which doesn't affect this process.